spec/requests/search_spec.rb

```diff
@@ -1,64 +1,70 @@
 # frozen_string_literal: true
-describe SearchController, type: :controller do
+
+describe 'Search', type: :request do
   let(:user) { create(:user) }
-  let(:user2) {  create(:user, email: 'bar@email.com') }
-  let(:user3) {  create(:user, email: 'foo@email.com', banned: true) }
+  let!(:user2) do
+    create(:user, email: 'bar@email.com',
+                  name: 'user 2 name')
+  end
+  let!(:user3) do
```

Comment:
Same here.

Reply:
Sure will do this change. Thanks for the feedback.